hikari/internal/fast_protocol.py

```diff
@@ -32,7 +32,7 @@
     from typing_extensions import Self
 
 _Protocol: FastProtocolChecking = NotImplemented
-_IGNORED_ATTRS = typing.EXCLUDED_ATTRIBUTES + ["__qualname__", "__slots__"]
+_IGNORED_ATTRS = frozenset(typing.EXCLUDED_ATTRIBUTES) | {"__qualname__", "__slots__"}
```

Comment:
`EXCLUDED_ATTRIBUTES` is a frozenset on 3.12 but a list on previous versions, so this seems not versions dependent